index.html

```diff
@@ -1,49 +1,219 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
-    
-    <link rel="stylesheet" href="./style.css">
-    
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css" />
     <title>Portfolio</title>
+  </head>
 
-</head>
-<body>
+  <body>
     <div class="container">
-       <header>
-         <div class="nav-menu">
-           <a class="logo" href="./index.html"> John Doe</a>
-           <nav class="menu"><img src="./svg/menu.svg" alt="Menu"></nav>
-         </div>
-       </header>
-       <section>
-       <div class="data">
-             <h1>
-              Hey There.
-              I’m  Ehsan
-             </h1>
-             <h2>
-              I’am a Software Developer
-             </h2>
-             <p>
-              I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-              don’t hestiate to contact me.
-             </p>
-             </div>
+      <div class="main">
+        <header>
+          <a class="logo" href="./index.html"> John Doe</a>
+          <nav><img src="./svg/menu.svg" alt="Menu" /></nav>
+        </header>
+        <section>
+          <h1>Hey There. I`m Ehsan</h1>
+          <h2>I`am a Software Developer</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don`t hestiate to contact me.
+          </p>
+          <ul class="social-media">
+            <li><img src="./svg/Vector1.svg" alt="" /></li>
+            <li><img src="./svg/github.svg" alt="" /></li>
+            <li><img src="./svg/linkedin.svg" alt="" /></li>
+            <li><img src="./svg/Vector3.svg" alt="" /></li>
+            <li><img src="./svg/twitter.svg" alt="" /></li>
+            <li><img src="./svg/Vector5.svg" alt="" /></li>
+          </ul>
+        </section>
+      </div>
+      <div class="work-section">
```

Comment:
- [ ] Kindly make sure to wrap the section of work-section in `section tag` instead of `div` for semantic purposes.